pallets/asset/src/lib.rs

```diff
@@ -753,31 +765,36 @@ decl_module! {
             let mut issued_in_this_round = Self::issued_in_funding_round(&ticker_round);
 
             // A round of per-investor checks
-            for i in 0..issue_asset_items.len() {
+            for IssueAssetItem { investor_did, value } in &issue_asset_items {
                 ensure!(
-                    Self::check_granularity(&ticker, issue_asset_items[i].value),
+                    Self::check_granularity(&ticker, *value),
                     Error::<T>::InvalidGranularity
                 );
                 let updated_total_supply = token
                     .total_supply
-                    .checked_add(&issue_asset_items[i].value)
+                    .checked_add(value)
                     .ok_or(Error::<T>::TotalSupplyOverflow)?;
                 ensure!(updated_total_supply <= MAX_SUPPLY.into(), Error::<T>::TotalSupplyAboveLimit);
 
-                current_balances.push(Self::balance(&ticker, &issue_asset_items[i].investor_did));
-                updated_balances.push(current_balances[i]
-                    .checked_add(&issue_asset_items[i].value)
-                    .ok_or(Error::<T>::BalanceOverflow)?);
+                let bals = Self::balance(&ticker, *investor_did);
+                current_total_balances.push(bals.total);
+                // No check since the total balance is always less than or equal to the total
```

Comment:
```suggestion
                // No check since the total balance is always <= to the total
```

Reply:
Any reason for your preference of symbols over English in comments?

Reply:
A nit, but using `<=` reads a bit quicker imo and requires less thinking for me than "less than or equal", but ymmv. :)

Reply:
Sure. Would you also like to replace the other 6 occurrences of "less than or equal to" and would you like to include the "to"?

There are also 8 occurrences of "greater than or equal to". What is your opinion about those?

Reply:
No opinion re. "to". For consistency, if you want to change to `<=` then ideally `>=` would be used also.

Reply:
Regarding "to", consider "A <= B" and "A <= to B". Which one you prefer?

Reply:
Slight preference for the former I guess, but very slight. :)

Reply:
<= is a binary relation that expands to "is less than or equal to". So it includes the "to".

I've done the substitution of <= and >=. Personally, I think it's not worth the bother.